.github/workflows/release-homebrew.yml

```diff
@@ -0,0 +1,41 @@
+name: Release Homebrew
+
+on:
+  workflow_call:
+  workflow_dispatch:
+    input:
+      notUsed:
+        type: string
+        description: Run this workflow from a "release-*" branch. This input does nothing.
+        required: false
```

Comment:
This is questionable, but could save someone time in the future.

Reply:
Is this input here just to provide the instruction to run the workflow on a release branch?

Reply:
It is. No other purpose.